lib/deaf_grandma.rb

```diff
@@ -14,14 +14,30 @@ def run!
     loop do
       user_input = get_user_input
       p speak(user_input)
+      if @bye_counter == 3 then
+        exit
+      end
     end
   end
 
 
   def speak(input)
-
-    #Implement your code here <<<<<<<<<
-
+    if input == input.upcase then
```

Comment:
`then` is optional in Ruby and typically unused. (It might be true that it is always unused.)